homeassistant/components/html5/issues.py

```diff
@@ -0,0 +1,42 @@
+"""Issues utility for HTML5."""
+
+import logging
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SUCCESSFUL_IMPORT_LOG = """Loading HTML5 push notification via configuration.yaml is deprecated,
+ it has been automatically imported. Once you have confirmed correct
+ operation, please remove HTML5 notification configuration from
+ configuration.yaml""".replace("\n", "")
+
+FAILED_IMPORT_LOG = """Loading HTML5 push notification via configuration.yaml is deprecated and
+ the automatic import has failed. Please remove the HTML5 notification configuration from configuration.yaml
+ and setup HTML5 from scratch via the UI.""".replace("\n", "")
+
+
+def create_issue(hass: HomeAssistant, import_success: bool):
```

Comment:
Please prefix functions that are safe to run in the event loop with `async_`

Reply:
Done